Game/MainScene.gd

```diff
@@ -22,7 +22,7 @@ func _init():
 	flagsCache = Flag.getFlags()
 
 func overridePC():
-	if(overridenPC != null):
+	if(is_instance_valid(overridenPC)):
```

Comment:
It's have a point. At least I recommend use for non-objects (array, int, string, etc.) use empty analog or set to specified value and check for it. It's help to track what happens in your code and catch some insidious bags.

I remember how I try guessing what value passes from GlobalRegistry.getCharacter() function in ArtworkPanel - because inside GlobalRegistry it's call function of other object in array.